libs/ledgerjs/packages/hw-app-elrond/src/Elrond.ts

```diff
@@ -111,9 +115,24 @@ export default class Elrond {
     ]);
   }
 
-  async signTransaction(path: string, message: string, usingHash: boolean): Promise<string> {
+  // kept 'usingHash' for compatibility reasons, siging by hash is the only supported way
+  async signTransaction(path: string, message: string, _usingHash?: boolean): Promise<string> {
```

Comment:
Can we already drop `_usingHash?` maybe?

Reply:
Let's double check with @bogdan-rosianu, that it's perfectly safe to assume that _"siging by hash is the only supported way"_.

Reply:
ok, dropped it.